### ISSUE TITLE
Config to remove the little clay icon above hotbar or add support for extended hotbar. 

### ISSUE THREAD

**Issue by TheShadyyOneYT** [Globox1997/EnvironmentZ#121]
### Make sure you are not opening a duplicate.

- [x] I have searched the issue tracker and did not find an issue describing my bug/feature.

### Description of your suggestion.

Currently I'm running into an issue where the clay icon above my hotbar is really obtrusive of my view, granted it it really small, but I don't see a purpose to keep it there when we have the thermometer already displayed on screen.  So adding a config to enable/disable it would be lovely, and/or compatible between extended hotbar, so the clay icon appears above the 2 hotbar and not in between them. https://imgur.com/a/cDlHCSh | Extended Hotbar mod: https://modrinth.com/mod/extended-hotbar

**Comment by Globox1997** [comment]
Are you talking about the body representing icon in the middle of the screen, a little higher than the hotbar?
If so, this is your body temperature. The thermometer is the environment temperature.
I could add a config option to position this icon manually (if I haven't done this yet)

**Comment by TheShadyyOneYT** [comment]
Yes that, exactly. The temperature for the body. Also, Is it meant to change colors? It always stays as the same color no matter where I am, even the same across dimensions. And Yeah adding some config would be nice. Just moving it up slightly or changing where it goes (assuming on a x and y plane) would be great. Anything like that or similar would be lovely, And thanks for the clarification that its your body temp, I thought it was but at the same time I was unsure due to the fact it doesn't change colors currently or does really anything when I use Environmentz. Thank you for your consideration of potentially adding a config, ~Cheers TheShadyy
